app/icon.py

```diff
@@ -77,6 +80,7 @@ def get_icon_filepath(self):
         Returns:
             A path leading to the file for this icon (can be an invalid one, do check its validity!)
         """
+        # TODO the service does only expect .png
```

Comment:
I discussed yesterday with @pakb 
This enforcement of .png extension is legacy, there were some PNGs which had no ending so it was enforced here.
Probably it is not necessary, that the serivce only returns PNGs. The viewer could most likely handle SVGs directly, right @pakb ? At least we should test this.

The BABS icons will have a pre-defined color, so there should and must not be a possibility to re-define the color. As for the scaling, we need to find a way to scale the SVGs. Probably some research needs to be done on this, there hopefully is a Python library for this.
Maybe the scaled SVGs need to be converted to another format (probably they'll become PNGs anyways) :wink: